src/autogluon/bench/frameworks/timeseries/timeseries_benchmark.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import subprocess
+import sys
+
+from autogluon.bench.frameworks.benchmark import Benchmark
+
+logger = logging.getLogger(__name__)
+
+
+class TimeSeriesBenchmark(Benchmark):
+    def setup(
+        self,
+        git_uri: str = "https://github.com/openml/automlbenchmark.git",
+        git_branch: str = "stable",
+        framework: str = "AutoGluon:stable",
+        user_dir: str = None,
+    ):
+        """Sets up the virtual environment for timeseries benchmark."""
+        setup_script_path = os.path.abspath(os.path.dirname(__file__)) + "/setup.sh"
+        command = [setup_script_path, git_uri, git_branch, self.benchmark_dir, framework]
+        if user_dir is not None:
+            command.append(user_dir)
+        result = subprocess.run(command)
```

Comment:
I think passing in `check=True` will catch the error directly so you don't need to check yourself

Reply:
I need to raise the error with `sys.exit(1)` to exit the container since otherwise it continues with the rest of the code.

Reply:
Isee